testsuite/integration/multinode/src/test/java/org/jboss/as/test/multinode/transaction/nooutbound/TransactionContextRemoteCallTestCase.java

```diff
@@ -85,15 +81,15 @@
     @Test
     @OperateOnDeployment(CLIENT_DEPLOYMENT)
     public void remotePlusHttpMandatoryBean() {
-        ProviderUrlData providerUrl = new ProviderUrlData("remote+http", url.getHost(), serverPort);
+        ProviderUrlData providerUrl = new ProviderUrlData("remote+http", TestSuiteEnvironment.getServerAddressNode1(), serverPort);
```

Comment:
This test now reads a system property so I think the set of permissions added to CLIENT_DEPLOYMENT in clientDeployment() needs to include a PropertyPermission.